packages/docs/package.json

```diff
@@ -37,6 +37,9 @@
   "devDependencies": {
     "@docusaurus/module-type-aliases": "2.0.0-beta.20",
     "@tsconfig/docusaurus": "1.0.5",
+    "docusaurus-plugin-typedoc": "^0.17.5",
```

Comment:
Minor, but I've been trying to maintain pinned versions of everything to keep cross-package versions more sane.  Prefer to drop the "^"

Reply:
yeah that's fine. This was automatic